config/buildspec_zero_code_change.yml

```diff
@@ -18,9 +18,8 @@ env:
 phases:
   install:
     commands:
-        -  . config/change_branch.sh
```

Comment:
why was this removed?

We may want to start build selectively. Say, I want to run TF build on certain commit/branch, 
Is above possible if you remove this.

Reply:
pip wheels are built from master branch. giving an option here to choose a different branch might cause inconsistencies.